### PR TITLE
update for 2023 analysis run

### LINKED ISSUE #8: Change README to reflect that buffering is included
The input data isn't buffered, instead this happens as part of the SQL

### PR DESCRIPTION
**Pull request by damonmcc**
resolves https://github.com/NYCPlanning/edm-overview/issues/897 (resolves DE part of it)
resolves #8

## changes
- move primary sql query out of `aue.sh` and into `calculate_intersections.sql`
- use an `INPUT_VERSION` envar to define which archived source data to use
- remove old (real) script outputs and commit test outputs
- save outputs csv files to subfolders named for the input data version
- add a script to install python packages already declared in `requirements.txt`
- add a devcontainer and related setup scripts

## notes
- most urgent goal was to rerun this analysis with new input data from GIS. that input data was archived to the `edm-data.recipes` database via data-library branch in https://github.com/NYCPlanning/db-data-library/pull/388
- latest outputs have been shared with GIS (from commits https://github.com/NYCPlanning/aue/commit/6d827e90bee80840be5f0c4ce1d47dc852b02ec4 and https://github.com/NYCPlanning/aue/pull/11/commits/6b7e92b2d967dc4ced57c0769142ceb4c65eb990)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,34 +1,36 @@
...
-We approached this problem by creating a matrix of possible co-existing establishments -- lots that can simultaneaously have establishments because they are further than 500ft apart. Simplifying the distance calculations to a binary flag makes our calculations much easier. 
+We approached this problem by creating a matrix of possible co-existing establishments -- lots that can simultaneously have establishments because they are further than 500ft apart. Simplifying the distance calculations to a binary flag makes our calculations much easier.
 
-### Data loading and preparation
+#### Data loading and preparation
 
 **Input data**:
 
-The input data for this analysis is the subset of MapPLUTO lots that meet the criteria, buffered to 500ft. See the image below for an example of what this data looks like.
+The input data for this analysis is the subset of MapPLUTO lots that meet the exclusion criteria. See the image below for an example of what this data looks like.
```
Comment (mbh329) [inline README.md]: Maybe we can add that this data comes from GIS and that it has to be archived via postgres into `EDM-DATA`?

File: example.env [hunk 2]
```diff
@@ -0,0 +1,2 @@
+RECIPE_ENGINE=
+INPUT_VERSION=
```
Comment (mbh329) [inline example.env:2]: Is the current INPUT_VERSION set to the weird `20230426_nom2m3`? If so, I think we should archive that again so it follows the conventions

File: bash/setup_devcontainer.sh [hunk 3]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+#
+# Sets up enviroment for dev in local devcontainer.
+set -e
+
+# Skip when called in a github action workflow
+if [[ ${CI} != "true" ]]; then
```
Comment (fvankrieken) [inline bash/setup_devcontainer.sh:7]: dev container shouldn't be running in CI, no?

File: example.env (hunk 2 above)
Comment (damonmcc) [reply example.env:2]: see reply to data library PR [here](https://github.com/NYCPlanning/db-data-library/pull/392#issuecomment-1526275685) (sorry it's a paragraph)

and INPUT_VERSION is only set in a `.env` file, which someone would have to create to run this. so it isn't set

File: bash/setup_devcontainer.sh (hunk 3 above)
Comment (damonmcc) [reply bash/setup_devcontainer.sh:7]: they actually can be (and maybe we'd wanna use them to run builds in CI sometimes?)

this script was copied from the template repo where the [dev container is run in CI](https://github.com/NYCPlanning/db-template-repo/blob/main/.github/workflows/tests_branch.yml#L51-L60) primarily to make sure it builds successfully. but we could potentially use dev containers to run production scripts to help unify local and CI environments

happy to take this out if it's more misleading than valuable

Comment (fvankrieken) [reply bash/setup_devcontainer.sh:7]: Nah that's fine!

File: README.md (hunk 1 above)
Comment (damonmcc) [reply README.md]: added in https://github.com/NYCPlanning/aue/pull/11/commits/648f7d1b1dec4b938983f44e8c09859f6655256a

**Review by fvankrieken — APPROVED** [review]

**Pull request merged**